src/styles/blocks/catalog.scss

```
@@ -17,6 +17,13 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: transform $card-element-transition,
```

usually the same transition duration is used across the whole project, so there's no need to specify the place where you use it. As variables are created for reusability - you should give them more generic names
```suggestion
    transition: transform $transition-duration,
```